base/rational.jl

```diff
@@ -279,7 +279,7 @@ function -(x::Rational{T}) where T<:Unsigned
     x
 end
 
-function +(x::Rational, y::Rational)
+function +(x::Rational, y::Rational)::Rational
     xp, yp = promote(x, y)
```

Comment:
```suggestion
    xp, yp = promote(x, y)::NTuple{2,Rational}
```
How about adding annotations here instead ?

Reply:
Yeah, seems reasonable.